File: python_programming/notes/Practices/time_of_day.py
#EH 6th Time of day

time=int(input("What is the hour in military time: "))

if time >= 0 and time <= 4:
    print("Good morning...")
elif time >= 5 and time <= 11:
    print("Good morning!")
elif time >11 and time <= 16:
    print("Good afternoon!")
elif time >=17 and time<=24:
    print("Good evening!")
elif time > 24:
    print("Hey you, enter a valid time!")
else:
    print("Good day!")

File: c_programming/practices/time_of_day.c
// EH 6th Time of Day

#include <stdio.h>

int main(void){
    int time;
    printf("What is the hour in military time: ");
    scanf("%d", &time);

    if(time >= 0 && time <= 4){
    printf("Good morning...");
    }else if(time >= 5 && time <= 11){
    printf("Good morning!");
    }else if(time >11 && time <= 16){
    printf("Good afternoon!");
    }else if (time >=17 && time<=24){
    printf("Good evening!");
    }else if(time > 24 || time < 0){
    printf("Hey you, enter a valid time!");
    }else{
    printf("Good day!");
    }
    return 0;
}